website/index.html

```diff
@@ -161,7 +161,7 @@ <h4>Custom Converters</h4>
             <section id="download">
               <h3>Download</h3>
               <p><a href="https://search.maven.org/remote_content?g=com.squareup.retrofit2&a=retrofit&v=LATEST" class="dl version-href">&darr; <span class="version-tag">Latest</span> JAR</a></p>
-              <p>The source code to the Retrofit, its samples, and this website is <a href="https://github.com/square/retrofit">available on GitHub</a>.</p>
+              <p>The source code to the Retrofit, its samples, and this website is <a href="https://github.com/square/retrofit">available on GitHub</a>.You can get retrofit's latest or history version JAR from <a href="https://github.com/square/retrofit/releases">retrofit release page</a> or <a href="https://mvnrepository.com/artifact/com.squareup.retrofit">maven central repository</a>.</p>
```

Comment:
The latest version is on Maven Central, not mvnrepository.com which isn’t an official site.

https://search.maven.org/artifact/com.squareup.retrofit2/retrofit

Reply:
Yes,I will modify this error,thanks.